ladybug_geometry/geometry2d/line.py

```diff
@@ -71,12 +76,24 @@ def length(self):
 
     def to_array(self):
         """ A nested list representing the two line endpoint coordinates.
-            Returns: 
+            Returns:
                 Nested tuples ((pt1.x, pt1.y), (pt2.x, pt2.y)), where
                 pt1 and pt2 represent the endpoints of the line segment.
         """
         return (self.p1.to_array(), self.p2.to_array())
 
+    @classmethod
+    def from_array(cls, line_array):
+        """ Consumes a nested list representing the two line endpoint coordinates
+        and returns a LineSegment2D.
+            Args:
+                Nested tuples ((pt1.x, pt1.y), (pt2.x, pt2.y)), where
+                pt1 and pt2 represent the endpoints of the line segment.
+            Returns:
+                LineSegment2D.
```

Comment:
I don't think you need to have a `Returns` docstring. If it's a classmethod, then it's evident that it returns an instance of the class.

Also, for the sake of organization, can you move this to be next to the other classmethods?